parsl/monitoring/monitoring.py

```diff
@@ -46,7 +46,8 @@ def __init__(self,
                  logging_endpoint: Optional[str] = None,
                  monitoring_debug: bool = False,
                  resource_monitoring_enabled: bool = True,
-                 resource_monitoring_interval: float = 30):  # in seconds
+                 resource_monitoring_interval: float = 30,
+                 file_provenance: bool = False):  # in seconds
```

Comment:
that comment belongs to the original line it was on!

Reply:
Yes it does.